Functions/CatalogItems/Remove-RsCatalogItem.ps1

```diff
@@ -4,75 +4,73 @@
 function Remove-RsCatalogItem
 {
     <#
-    .SYNOPSIS
-        This function removes an item from the Report Server Catalog.
-
-    .DESCRIPTION
-        This function removes an item from the Report Server Catalog. 
-
-    .PARAMETER ReportServerUri
-        Specify the Report Server URL to your SQL Server Reporting Services Instance.
-
-    .PARAMETER ReportServerCredentials
-        Specify the credentials to use when connecting to your SQL Server Reporting Services Instance.
-
-    .PARAMETER Proxy
-        Specify the Proxy to use when communicating with Reporting Services server. If Proxy is not specified, connection to Report Server will be created using ReportServerUri, ReportServerUsername and ReportServerPassword.
+        .SYNOPSIS
+            This script creates a new data source on Report Server.
```

Comment:
Seems like a copy and paste error, please put 
"This function removes an item from the Report Server Catalog."